src/components/GroupSelector.tsx

```diff
@@ -1,29 +1,27 @@
 import React from 'react';
 import { Select, Option } from '@material-tailwind/react';
-import { SetterOrUpdater } from 'recoil';
-
-interface GroupSelectorProps {
-  selectedGroup: string;
-  setSelectedGroup: SetterOrUpdater<string>;
-}
+import { useNavigate, useParams } from 'react-router-dom';
 
 const GROUPS = [
-  { id: 1, title: '부산대학교' },
-  { id: 2, title: '이삼정보고' },
-  { id: 3, title: '카카오테크캠퍼스' },
+  { id: 1, label: '부산대학교', value: 'pusan' },
+  { id: 2, label: '이삼정보고', value: 'isam' },
+  { id: 3, label: '카카오테크캠퍼스', value: 'kakao' },
 ];
 
-const GroupSelector = ({ selectedGroup, setSelectedGroup }: GroupSelectorProps) => {
-  const handleClick = (title: string) => {
-    setSelectedGroup(title);
+const GroupSelector = () => {
+  const { groupName } = useParams();
+  const navigate = useNavigate();
+
+  const handleChange = (value: string | undefined) => {
```

Comment:
`undefined` 가능한 옵셔널 값이면 아래와 같이 처리 가능할 것 같아요!

```suggestion
  const handleChange = (value?: string) => {
```

Reply:
오 꿀팁 감사합니다 ㅎㅎ수정하겠습니다